.github/workflows/ci.yaml

```diff
@@ -88,7 +88,7 @@ jobs:
         export PATH=`pwd`:$PATH
         cp node_modules/geckodriver/geckodriver.exe .
         yarn test-browser --headless-firefox --timeout ${{ env.browserTestTimeout }}
-      if: matrix.os == 'windows-latest'
+      if: matrix.os == 'windows-2019'
```

Comment:
you also need to change the if condition for the Firefox Windows test

Reply:
i believe windows-latest is an alias to windows-2019